packages/runtime-common/router.ts

```diff
@@ -72,7 +72,7 @@ export class Router {
       let routeRegExp = new RegExp(`^${route.replace("/", "\\/")}$`);
       if (routeRegExp.test(requestPath)) {
         try {
-          return await handler(request);
+          return await handler(request, response);
```

Comment:
we actually should create the response here. This should be the only place in the codebase where a response is created.

Reply:
and notably, this is also where we can add the `vary` header